packages/common/src/compiler/translator.ts

```diff
@@ -300,16 +300,17 @@ export const convertTable = (
                 extraDimensions = intervals.reduce(
                     (acc, interval) => ({
                         ...acc,
-                        [`${column.name}_${interval}`]: convertDimension(
-                            index,
-                            adapterType,
-                            model,
-                            tableLabel,
-                            column,
-                            undefined,
-                            interval,
-                            startOfWeek,
-                        ),
+                        [`${column.name}_${interval.toLowerCase()}`]:
```

Comment:
Unnoticed bug where the key of time interval dimensions wouldn't match the `name` property.

`order_date_DAY` vs `order_date_day`

<img width="461" alt="Screenshot 2023-12-07 at 13 26 08" src="https://github.com/lightdash/lightdash/assets/9117144/98f63316-c483-47dd-b71e-0ca127b67085">

Reply:
will this affect existing explores ? 
So if someone is trying to filter use date zoom on a explore that hasn't been updated, it will not work, right ?